app/models/organization.js

```diff
@@ -0,0 +1,29 @@
+import mongoose from 'mongoose';
+
+var Schema = mongoose.Schema;
+
+var OrgSchema = new Schema({
+  name: {
+    type: String,
+    required: true
+  },
+  email: {
+    type: String,
+    required: true
+  },
+  description: {
+    type: String,
+    required: true
+  },
+  founded: {
+    type: Date,
+    default: Date.now
+  },
+  Founder: {
```

Comment:
Can we extend this to allow for multiple founders? (Hint: lets create another model for a founder and then use an array of instances here).

Lets also continue to use lowercase field names so `Founder` => `founder`